chrome/_locales/pt_BR/messages.json

```diff
@@ -0,0 +1,28 @@
+{
+    "extDescription": {
+      "message": "Bloquear anúncios nas transmissões da Twitch.tv.",
+      "description": "A descrição da extensão."
+    },
+    "toggleAdblocker": {
+        "message": "Bloquear anúncios"
+    },
+    "toggleBlockingAds": {
+        "message": "Aviso de \"Bloqueando anúncios\"",
```

Comment:
It's not entirely a warning message, "Mensagem de \"Bloqueando anúncios\" Off / On" is my suggestion